src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -108,4 +144,40 @@ private static String launchAndJoinCommand(Launcher launcher, List<String> args)
 
     return cmdLogStream.toString(CHARSET);
   }
+
+  /**
+   * Using the kubectl cli tool as the api client for the caller, this method unmarshalls the json
+   * output of the cli to a JSON Object.
+   *
+   * @param kind - the kind of kubernetes Object
+   * @param name - the name of the kubernetes Object
+   * @return a JSON object
+   */
+  public Object getObject(String kind, String name) throws IOException, InterruptedException {
+    String json = runKubectlCommand("get", ImmutableList.<String>of(kind, name, "-o", "json"));
+    return Configuration.defaultConfiguration().jsonProvider().parse(json);
+  }
+
+  /**
+   * @param kind - kubernetes object kind
+   * @param labels - the key-vale labels set represented as a map
+   * @return a list of JSON objects
+   * @throws IOException
```

Comment:
Header is missing method description.
Also \@throws annotations are missing descriptions.